router/pkg/config/config.go

```diff
@@ -232,8 +232,7 @@ type RateLimitConfiguration struct {
 }
 
 type RedisConfiguration struct {
-	Addr      string `yaml:"addr" default:"localhost:6379" envconfig:"REDIS_ADDR"`
-	Password  string `yaml:"password,omitempty" envconfig:"REDIS_PASSWORD"`
+	Url       string `yaml:"url,omitempty" default:"redis://localhost:6379" envconfig:"REDIS_URL"`
```

Comment:
I propose to prefix all environment variables with `RATE_LIMIT_` to make the position clear in the configuration.

```
RATE_LIMIT_REDIS_URL
RATE_LIMIT_REDIS_KEY_PREFIX
```